plugins/woocommerce/src/Internal/Admin/ReviewsListTable.php

```diff
@@ -5,13 +5,65 @@
 
 namespace Automattic\WooCommerce\Internal\Admin;
 
+use WP_Comment;
+use WP_Comments_List_Table;
 use WP_List_Table;
 
 /**
  * Handles the Product Reviews page.
  */
 class ReviewsListTable extends WP_List_Table {
 
+	/**
+	 * Memoization flag to determine if the current user can edit the current review.
+	 *
+	 * @var bool
+	 */
+	private $current_user_can_edit = false;
+
+	/**
+	 * Prepares reviews for display.
+	 *
+	 * @return void
+	 */
+	public function prepare_items() {
+
+		$comments = get_comments(
+			[
+				'post_type' => 'product',
+			]
+		);
+
+		update_comment_cache( $comments );
+
+		$this->items = $comments;
+	}
+
+	/**
+	 * Render a single row HTML.
+	 *
+	 * @param WP_Comment $item Review or reply being rendered.
+	 * @return void
+	 */
+	public function single_row( $item ) {
+		global $post, $comment;
+
+		// Overrides the comment global for properly rendering rows.
+		$comment           = $item; // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
+		$the_comment_class = wp_get_comment_status( $comment->comment_ID );
```

Comment:
```suggestion
		$the_comment_class = wp_get_comment_status( $comment->comment_ID ) ?: '';
```

`wp_get_comment_status()` can return `false` on failure, and `get_comment_class()` expects the first parameter to be a string. So this suggestion just helps ensure we always have a string.

Could also just cast it to a string instead.